.github/workflows/canister-tests.yml

```diff
@@ -391,6 +390,20 @@ jobs:
         env:
           RUST_BACKTRACE: 1
 
+  test-canisters-script:
```

Comment:
This job takes more than 10 minutes, so it is immediately the slowest thing in the whole pipeline. I would not add it without any caching.

For reference, the normal pipeline with caching takes ~6m 30s.

Reply:
I added the `--no-build` parameter and used the built wasms. It takes 6'30" now. Is this reasonable?

It depends on a few other jobs though. Therefore, the whole CI is probably slower now.

I could add pocket-ic and the other dependencies into a cache and use it as well. Maybe in another PR?